README.md

```diff
@@ -1,4 +1,11 @@
-# [![Flutter logo][]][flutter.dev]
+<a href="https://flutter.dev/">
+  <h1 align="center">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://storage.googleapis.com/cms-storage-bucket/6e19fee6b47b36ca613f.png">
+      <img alt="The Flutter logo" src="https://storage.googleapis.com/cms-storage-bucket/c823e53b3a1a7b0d36a9.png">
```

Comment:
the alt text here should probably just be "Flutter" (or nothing), since the image isn't conveying "The Flutter logo" it's conveying "Flutter" (or arguably nothing, as a decorative emphasis or something).

Reply:
(yes this was probably wrong before too)